lib/instance.js

```diff
@@ -482,6 +482,8 @@ Instance.prototype._setInclude = function(key, value, options) {
  *
  * @param {Object} [options]
  * @param {string[]} [options.fields] An optional array of strings, representing database columns. If fields is provided, only those columns will be validated and saved.
+ * @param {string[]} [options.where] An optional object with where conditions.
```

Comment:
Ah, i looked at the code before reading the PR description, and I fully realize how this can help UoW. But I'm still not sure i like it